src/ducks/transactions/actions/KonnectorAction.jsx

```diff
@@ -62,10 +62,12 @@ const Component = ({
   if (isModalItem) {
     return (
       <TransactionModalRow
-        text={label}
         iconLeft="plus"
         onClick={() => open(url)}
-      />
+        style={{ color: palette.dodgerBlue }}
```

Comment:
Same question for the { color: palette.dodgerBlue } class ?

Would you want utility classes for colors ?